WordPress/Classes/System/SplitViewRootPresenter+Reader.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+import UIKit
+
+class ReaderSplitViewContent: SplitViewDisplayable {
+    let sidebar: ReaderSidebarViewController
+    let sidebarNavigationController: UINavigationController
+    var content: UINavigationController
```

Comment:
If `secondary` becomes non-optional (my other comment), there is no need for a separate `content` . It can be a it confusing that it has both. In the same vein, `sidebarNavigationController` could also be merged with `supplimentary`, there appears to be no need for a separate property.